easy_pil/canvas.py

```diff
@@ -27,15 +27,15 @@ class Canvas:
 
     def __init__(
         self,
-        size: Tuple[float, float] = None,
-        width: float = None,
-        height: float = None,
-        color: Color = None,
+        size: Optional[Tuple[int, int]] = None,
+        width: int = 0,
+        height: int = 0,
+        color: Color = 0,
     ) -> None:
-        if not size and not width and not height:
+        if not (size or (width and height)):

```

Comment:
For some reason this actually causes the canvas test to fail
maybe a better approach is 

```py
if not any([size, width, height]):
    raise ValueError("size, width, and height cannot all be None")

if size:
    self.size = size
elif width and height:
    self.size = (width, height)
else:
    raise ValueError("width, and height both must be provided")
```

Reply:
ig this kind of checking will introduce bugs in whole code base

Reply:
lemme check the tests again